bvm/ballerina-core/src/main/java/org/ballerinalang/util/transactions/TransactionUtils.java

```diff
@@ -70,7 +83,7 @@ public static boolean isInitiator(Strand ctx, String globalTransactionId,
 
     private static void checkTransactionCoordinatorError(BValue value, Strand ctx, String errMsg) {
         if (value.getType().getTag() == TypeTags.ERROR_TAG) {
-            throw new BallerinaException(errMsg + ((BError) value).details);
+            throw new BallerinaException(errMsg + ((BError) value).reason);
```

Comment:
ctx is unused parameter?

Reply:
Will do, shall we leave the unused method isInitiator there, as it is the java proxy method to transaction coordinator's isInitiator method and if we remove it, future us, might forget that java can access that method.